libethash-cuda/ethash_cuda_miner_kernel_globals.h

```diff
@@ -6,3 +6,10 @@ __constant__ uint32_t d_light_size;
 __constant__ hash64_t* d_light;
 __constant__ hash32_t d_header;
 __constant__ uint64_t d_target;
+
+#if (__CUDACC_VER_MAJOR__ > 8)
+#define __shfls(w, x,y,z)               __shfl_sync((w),(x),(y),(z))
```

Comment:
Can you name it `SHFL` to indicate it is a macro?

Reply:
Yes, it is Macro .
If you want to then change it your coding rule.
And, Originally, I did 2 PR,
But, when CI/Travis, another commit to my repo. it is automatically added.
So, I'm not well do github PR.
Merge and revert last commit is also good way.

Reply:
you have to separate two PRs into another branches like as following
~~~
$ git checkout -b fix-win32-icon master
$ git checkout -b fix-cuda-8-shfl master
~~~
then strash some commits like as
~~~
$ git checkout fix-cuda-9-shfl
$ git reset 32f32da --hard
$ git push -f fix-cuda-9-shfl origin # or your remote repo
~~~